docs/contributing-to-airbyte/README.md

```diff
@@ -13,9 +13,9 @@ Before getting started, please review Airbyte's Code of Conduct. Everyone intera
 ## Code Contributions
 
 Most of the issues that are open for contributions are tagged with [`good-first-issue`](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue) or [`help-welcome`](https://github.com/airbytehq/airbyte/labels/help-welcome).
-A great place to start looking will be our GitHub projects for [**Community Connector Issues**](https://github.com/orgs/airbytehq/projects/50)
+A great place to start looking will be our GitHub projects for [**Marketplace Connector Issues**](https://github.com/orgs/airbytehq/projects/50)
```

Comment:
```suggestion
```

This link is to a project where we have _some_ but not all relevant issues, and a LOT of issues are in the closed repository. Until @marcosmarxm and I fix it, let's remove this link. I feel pretty strongly about this.